src/Components/Web.JS/src/DomWrapper.ts

```diff
@@ -35,3 +42,23 @@ function focusBySelector(selector: string): void {
     element.focus();
   }
 }
+
+function elementExists(identifier : string) : boolean {
+  let element : HTMLElement | null = null;
+  
+  element = document.getElementById(identifier);
+  
+  if (!element) {
+
+    let elements = document.getElementsByName(identifier);
+    if (elements.length > 0) {
+      element = elements[0];     
+    }
+  }
+  
+  if (element) {
+    return true;
+  }
+
+  return false;
```

Comment:
TypeScript/JavaScript is a pretty forgiving language when it comes to null handling, so this could all be reduced to:

```suggestion
  const element = document.getElementById(identifier)
      || document.getElementsByName(identifier)[0];
  return !!element;
```

Sidenote: we generally prefer `const` over `let` whenever possible, as this indicates that the variable won't be overwritten later. It's quite different to the meaning of `const` in C#.